package.json

```diff
@@ -84,7 +85,7 @@
     "browserstack-local": "^1.4.8",
     "chai": "^4.3.4",
     "chai-as-promised": "^7.1.1",
-    "chromedriver": "^94.0.0",
+    "chromedriver": "^96.0.0",
```

Comment:
Ik merk dat je pr bamboo build rood is, waarschijnlijk hier om.

Reply:
Heb het terug aangepast. Zometeen de build es checken.